solidity/test/integration/ConnextVestingWallet.t.sol

```diff
@@ -70,27 +70,19 @@ contract UnitConnextVestingWallet is Test, Constants {
    *  After the end of the unlocking period: 13 ether tokens
    */
   function test_UnlockedAtTimestamp() public {
-    assertEq(_connextVestingWallet.vestedAmount(NEXT_TOKEN_ADDRESS, _connextTokenLaunch), 0);
-    assertEq(_connextVestingWallet.vestedAmount(NEXT_TOKEN_ADDRESS, _firstMilestoneTimestamp - 1), 0);
+    assertEq(_connextVestingWallet.vestedAmount(_connextTokenLaunch), 0);
+    assertEq(_connextVestingWallet.vestedAmount(_firstMilestoneTimestamp - 1), 0);
 
-    assertEq(_connextVestingWallet.vestedAmount(NEXT_TOKEN_ADDRESS, _firstMilestoneTimestamp), 1 ether);
+    assertEq(_connextVestingWallet.vestedAmount(_firstMilestoneTimestamp), 1 ether);
 
-    assertApproxEqAbs(
-      _connextVestingWallet.vestedAmount(NEXT_TOKEN_ADDRESS, _firstMilestoneTimestamp + MONTH), 2 ether, MAX_DELTA
-    );
+    assertApproxEqAbs(_connextVestingWallet.vestedAmount(_firstMilestoneTimestamp + MONTH), 2 ether, MAX_DELTA);
```

Comment:
Why MAX_DELTA is so high? (1 ether)

Reply:
> Why MAX_DELTA is so high? (1 ether) I would use just `assertEq` on this case and all this file cases

mostly because we use approximate datatimes, like 30 days = 1 month, but 1 month also = 1/12 year, which is 365 days. So its many

Reply:
But in this case I mean.
`assertApproxEqAbs(_connextVestingWallet.vestedAmount(_firstMilestoneTimestamp + MONTH), 2 ether, MAX_DELTA);`

You are comparing the vestedAmount, to 2 ether. With a delta of 1 ether. In this specific case it is wrong